integration/spark/spark3/build.gradle

```diff
@@ -40,7 +40,6 @@ repositories {
 archivesBaseName='openlineage-spark-spark3'
 
 ext {
-    bigqueryVersion = '0.21.1'
```

Comment:
bigquery dependencies are not needed for `spark2`, `spark3` and `spark32` subprojects. Instead of upgrading, I remove them,